.ci/scripts/test_wheel_package_qnn.sh

```diff
@@ -0,0 +1,155 @@
+    compile_spec = generate_qnn_executorch_compiler_spec(
+        soc_model=get_soc_to_chipset_map()[args.soc],
+        backend_options=backend_options,
+    )
+    delegated_program = to_edge_transform_and_lower_to_qnn(m, example_inputs, compile_spec)
+    executorch_program = delegated_program.to_executorch(
+        config=ExecutorchBackendConfig(extract_delegate_segments=False)
+    )
+    save_pte_program(executorch_program, "linear", args.output_folder)
+
+if __name__ == "__main__":
+    main()
+EOF
+
+# ----------------------------
+# Wheel build and .so checks
+# ----------------------------
+echo "=== Building Wheel Package ==="
+EXECUTORCH_BUILDING_WHEEL=1 python setup.py bdist_wheel
+
+unset EXECUTORCH_BUILDING_WHEEL
+
+WHEEL_FILE=$(ls dist/*.whl | head -n 1)
+echo "Found wheel: $WHEEL_FILE"
+
+# ----------------------------
+# Check wheel does NOT contain qualcomm/sdk
+# ----------------------------
+echo "Checking wheel does not contain qualcomm/sdk..."
+SDK_FILES=$(unzip -l "$WHEEL_FILE" | awk '{print $4}' | grep "executorch/backends/qualcomm/sdk" || true)
+if [ -n "$SDK_FILES" ]; then
+    echo "ERROR: Wheel package contains unexpected qualcomm/sdk files:"
+    echo "$SDK_FILES"
+    exit 1
+else
+    echo "OK: No qualcomm/sdk files found in wheel"
+fi
+
+# ----------------------------
+# Conda environment setup & tests
+# ----------------------------
+TEMP_ENV_DIR=$(mktemp -d)
+echo "Using temporary directory: $TEMP_ENV_DIR"
+
+conda create -y -p "$TEMP_ENV_DIR/env" python=3.10
+conda run -p "$TEMP_ENV_DIR/env" pip install "$WHEEL_FILE"
+
+conda run -p "$TEMP_ENV_DIR/env" pip install torch=="2.9.0.dev20250801" --index-url "https://download.pytorch.org/whl/nightly/cpu"
+conda run -p "$TEMP_ENV_DIR/env" pip install --pre torchao --index-url "https://download.pytorch.org/whl/nightly/cpu"
+
+# ----------------------------
+# Check .so files in the wheel
+# ----------------------------
+echo "Checking for .so files inside the wheel..."
+WHEEL_SO_FILES=$(unzip -l "$WHEEL_FILE" | awk '{print $4}' | grep "executorch/backends/qualcomm/python" || true)
+if [ -z "$WHEEL_SO_FILES" ]; then
+    echo "WARNING: No .so files found in wheel under executorch/backends/qualcomm/python"
+else
+    echo "Wheel contains the following .so files:"
+    echo "$WHEEL_SO_FILES"
+fi
+
+# ----------------------------
+# Check installed .so files in conda env
+# ----------------------------
+echo "Checking installed executorch/backends/qualcomm/python contents in conda env..."
+ENV_SO_DIR="$TEMP_ENV_DIR/env/lib/python3.10/site-packages/executorch/backends/qualcomm/python"
+ls -l "$ENV_SO_DIR" || echo "Folder does not exist!"
+
+# Run import tests
+conda run -p "$TEMP_ENV_DIR/env" python -c "import executorch; print('executorch imported successfully')"
+conda run -p "$TEMP_ENV_DIR/env" python -c "import executorch.backends.qualcomm; print('executorch.backends.qualcomm imported successfully')"
```

Comment:
is it worth checking if the so files are downloaded and installed? Or the script running would suffice